cetera-http/src/main/scala/com/socrata/cetera/search/UserClient.scala

```
@@ -20,16 +23,54 @@ class UserClient(esClient: ElasticSearchClient, indexAliasName: String) {
     EsUser(res.getSourceAsString)
   }
 
+  // NOTE: this should be bigger than number of users w/ assets on a given domain
+  val OwnerAggregationSize = 1000
+
+  def fetchDomainOwnerIds(
+      domain: Option[Domain],
+      authorizedUser: Option[AuthedUser])
```

Right, but for posterity, as we discussed earlier, we should have a higher level check in the service so that we can fail fast.